src/main/java/duke/exception/IncompleteCommandError.java

```diff
@@ -0,0 +1,22 @@
+package duke.exception;
+
+public class IncompleteCommandError extends UnknownCommandException {
```

Comment:
Regarding packaging exceptions into a package:

I think this stackoverflow post would explain it better than me:
(Examples: java.**io**.IOException, java.**sql**.SQLException)
https://stackoverflow.com/questions/33480666/java-custom-exceptions-package-convention